#returns the current flow over the half bridge

int32 current_hal_bridge_1
int32 current_hal_bridge_2

